pulpcore.te

```diff
@@ -130,8 +130,8 @@ libs_exec_ldconfig(pulpcore_server_t)
 # Needed for systemd Type=notify support
 init_write_pid_socket(pulpcore_server_t)
 kernel_dgram_send(pulpcore_server_t)
-allow pulpcore_server_t init_var_run_t:unix_dgram_socket { connect create sendto };
-allow kernel_t init_var_run_t:unix_dgram_socket { connect create sendto };
+allow pulpcore_server_t init_var_run_t:unix_dgram_socket { connect create };
```

Comment:
I wonder if we need to label the systemd unit file to get it in the correct domain. We do so in foreman-selinux:

https://github.com/theforeman/foreman-selinux/blob/58f9b9b384d9a12338dfa7700b93c1d53e1e143e/foreman.te#L112-L113

Then we use `self:unix_dgram_socket`:
https://github.com/theforeman/foreman-selinux/blob/58f9b9b384d9a12338dfa7700b93c1d53e1e143e/foreman.te#L237

Reply:
This appears to be the good practice indeed: `ll -Z /usr/lib/systemd/system/`

Reply:
After reviewing some unit files, I suspect the purpose of having a unit type  is to help other binaries, such as an ExecStartPre binary, to work.

It seems most unit files under /usr/lib/systemd/system with an `ExecStartPre=` line have their own type.

Reply:
That may be a reason too, but the official reason is that it enables one service to control another. If we wanted foreman to be able to stop/start/restart pulp for example, then foreman would need access to the label on the pulp unit file:
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/selinux_users_and_administrators_guide/chap-security-enhanced_linux-systemd_access_control

So it is a good idea, but I will do it in a separate PR.

Reply:
This one should not be needed.